### PR TITLE
python3Packages.kaggle: 1.5.6 -> 1.5.9

### PR DESCRIPTION
**Pull request by guillaumecherel**
###### Motivation for this change

Upgrade the kaggle command line API to its latest version.

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [X] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/development/python-modules/kaggle/default.nix [hunk 1]
```diff
@@ -5,18 +5,20 @@
...
 
 buildPythonPackage rec {
   pname = "kaggle";
-  version = "1.5.6";
+  version = "1.5.9";
 
   src = fetchPypi {
     inherit pname version;
-    sha256 = "0f5qrkgklcpgbwncrif7aw4f86dychqplh7k3f4rljwnr9yhjb1w";
+    # sha256 = "0f5qrkgklcpgbwncrif7aw4f86dychqplh7k3f4rljwnr9yhjb1w";
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/kaggle/default.nix:20]:
```suggestion
```

**Comment by cdepillabout** [comment]
@guillaumecherel Do you want to add yourself as a maintainer for kaggle?

**Comment by guillaumecherel** [comment]
I'd rather not, I'm not a regular user.

File: maintainers/maintainer-list.nix [hunk 2]
```diff
@@ -3341,6 +3341,12 @@
     githubId = 1882000;
     name = "Guillaume Desforges";
   };
+  guillaumecherel = {
+    email = "guillaume.cherel@cosimus.com";
+    github = "guillaumecherel";
+    githubId = 6168820;
+    name = "Guillaume Chérel";
+  };
   guillaumekoenig = {
```
Comment (jonringer) [inline maintainers/maintainer-list.nix:3350]: Please make this into a separate commit:

```
maintainers: add <name>
```

**Review by jonringer — COMMENTED** [review]
this should be 3 commits

```
maintainers: add <name>
python3Packages.slugify: init at <version>
python3Packages.kaggle: 1.5.6 -> 1.5.9
```

`git rebase -i` is a good tool for this, I created a small video demonstrating it's use [here](https://www.youtube.com/watch?v=OR8Q-4PLalc). A more indepth text tutorial can be found [here](https://www.atlassian.com/git/tutorials/rewriting-history/git-rebase)

File: maintainers/maintainer-list.nix (hunk 2 above)
Comment (SuperSandro2000) [reply maintainers/maintainer-list.nix:3350]: Nitpick: commit messages should not end with a dot.

File: pkgs/development/python-modules/slugify/default.nix [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+      inherit pname version;
+      sha256 = "0k2y4cc0zkh3lzsifdx5hidbpf36cwqbps1wdx9lfs8s3k0kqw65";
+    };
+
+    meta = with stdenv.lib; {
+      homepage = "https://github.com/zacharyvoase/slugify";
+      description = "A generic slugifier utility, inspired by Django’s slugify template filter.";
+      license = licenses.unlicense;
+      platforms = platforms.all;
+      maintainers = with maintainers; [  ];
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/slugify/default.nix:17]: Can you add yourself here, too?

File: pkgs/development/python-modules/slugify/default.nix [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+    src = fetchPypi {
+      inherit pname version;
+      sha256 = "0k2y4cc0zkh3lzsifdx5hidbpf36cwqbps1wdx9lfs8s3k0kqw65";
+    };
+
+    meta = with stdenv.lib; {
+      homepage = "https://github.com/zacharyvoase/slugify";
+      description = "A generic slugifier utility, inspired by Django’s slugify template filter.";
+      license = licenses.unlicense;
+      platforms = platforms.all;
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/slugify/default.nix:16]: That is automatically set and only required if not all is supported.

File: pkgs/development/python-modules/slugify/default.nix [hunk 5]
```diff
@@ -0,0 +1,19 @@
+{ stdenv, fetchPypi, buildPythonPackage, python, text-unidecode }:
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/slugify/default.nix:1]:
```suggestion
{ stdenv, fetchPypi, buildPythonPackage }:
```

**Review by SuperSandro2000 — CHANGES_REQUESTED** [review]

**Comment by SuperSandro2000** [comment]
Master already contains the update.
